MatrixKit/Models/Contact/MXKContactManager.m

```diff
@@ -1579,16 +1590,23 @@ - (NSDictionary*)fetchCachedMatrixContacts
         {
             NSData* filecontent = [NSData dataWithContentsOfFile:dataFilePath options:(NSDataReadingMappedAlways | NSDataReadingUncached) error:nil];
             
-            NSKeyedUnarchiver *decoder = [[NSKeyedUnarchiver alloc] initForReadingWithData:filecontent];
-            
-            id object = [decoder decodeObjectForKey:@"matrixContactByContactID"];
-            
-            if ([object isKindOfClass:[NSDictionary class]])
-            {
-                matrixContactByContactID = object;
+            NSError *error = nil;
+            filecontent = [self decryptData:filecontent error:&error fileName:matrixContactsFile];
+
+            if (!error) {
```

Comment:
We use a line break before each `{}` block in obj-c code. We need to keep it for consistency.

Reply:
This comment applies as many times as required in this PR :)